docs/pages/eas/workflows/pre-packaged-jobs.mdx

```diff
@@ -0,0 +1,812 @@
+      API_URL: https://api.example.com
+    params:
+      platform: android
+      profile: production
+```
+
+</Collapsible>
+
+<Collapsible summary="Build with different profiles">
+
+This workflow creates two different Android builds using different profiles - one for internal distribution and one for store submission.
+
+```yaml .eas/workflows/build-profiles.yml
+name: Build with different profiles
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  build_android_internal:
+    name: Build Android Internal
+    type: build
+    params:
+      platform: android
+      profile: internal
+
+  build_android_store:
+    name: Build Android Store
+    type: build
+    params:
+      platform: android
+      profile: store
+```
+
+</Collapsible>
+
+## Deploy
+
+Deploy your application using [EAS Hosting](/eas/hosting/introduction).
+
+### Prerequisites
+
+To deploy your application using EAS Hosting, you'll need to set up your project. See [Get Started with EAS Hosting](/eas/hosting/get-started/#prerequisites) for more information.
+
+### Syntax
+
+```yaml
+jobs:
+  deploy_web:
+    type: deploy
+    params:
+      alias: string # optional
+      prod: boolean # optional
+```
+
+#### Parameters
+
+You can pass the following parameters into the `params` list:
+
+| Parameter | Type    | Description                                                                        |
+| --------- | ------- | ---------------------------------------------------------------------------------- |
+| alias     | string  | Optional. The [alias](/eas/hosting/deployments-and-aliases/#aliases) to deploy to. |
+| prod      | boolean | Optional. Whether to deploy to production.                                         |
+
+### Examples
+
+Here are some practical examples of using the deploy job:
+
+<Collapsible summary="Basic deployment to production">
+
+This workflow deploys your application to production using EAS Hosting.
+
+```yaml .eas/workflows/deploy-basic.yml
+name: Basic Deployment
+
+jobs:
+  deploy:
+    name: Deploy to Production
+    type: deploy
+    params:
+      prod: true
+```
+
+</Collapsible>
+
+<Collapsible summary="Deployment with custom alias">
+
+This workflow deploys your application to a custom alias in production.
+
+```yaml .eas/workflows/deploy-alias.yml
+name: Deployment with Alias
+
+jobs:
+  deploy:
+    name: Deploy with Alias
+    type: deploy
+    params:
+      alias: my-custom-alias
+      prod: true
+```
+
+</Collapsible>
+
+## Fingerprint
+
+Calculates a fingerprint of your project.
+
+> **Note:** This job type only supports [CNG (managed)](/workflow/continuous-native-generation/) workflows. If you commit your **android** or **ios** directories, the fingerprint job won't work.
+
+### Syntax
+
+```yaml
+jobs:
+  fingerprint:
+    type: fingerprint
+    params:
+      environment: production | preview | development # optional, defaults to production
+      env: # optional
+        APP_VARIANT: staging # just an example
```

Comment:
Note: [soon](https://github.com/expo/universe/pull/19535) we'll allow providing `environment` for every job which is important for `${{ env.… }}` interpolations.